src/main/java/frc/robot/Robot.java

```diff
@@ -193,6 +198,12 @@ public void autonomousPeriodic() {
      */
     @Override
     public void teleopPeriodic() {
+        //If motor current limiting is active, trigger driver feedback (notify driver that drivetrain power is reduced).
+        if (DriveBase.isMotorLimitActive() == true) {
+            //TODO: Put driver feedback code here. Consult with drive team's preferences.
+        } else {
+            //Disable current limit driver feedback here.
```

Comment:
Should we be disabling here?

Reply:
Yes, driver feedback should be turned off once the current limiter stops doing its job.
I'll rewrite this comment to make this more clear.

Reply:
I mean that I would like to see some code here instead of a comment, even if it is a best guess.

Reply:
OK, I'll fix that in my next commit.